R/type-date-time.R

```diff
@@ -427,6 +439,37 @@ vec_arith.difftime.Date <- function(op, x, y, ...) {
     stop_incompatible_op(op, x, y)
   )
 }
+#' @method vec_arith.POSIXct difftime
+#' @export
+vec_arith.POSIXct.difftime <- function(op, x, y, ...) {
+  y <- vec_cast(y, new_duration(units = "secs"))
+
+  switch(op,
+         `+` = vec_restore(vec_arith_base(op, x, y), x),
```

Comment:
Indenting

Reply:
I hate that RStudio does this automatically